test/tools/libtesteth/TestHelper.cpp

```diff
@@ -242,21 +243,31 @@ void replaceLLLinState(json_spirit::mObject& _o)
 	}
 }
 
-std::vector<boost::filesystem::path> getJsonFiles(boost::filesystem::path const& _dirPath, std::string const& _particularFile)
+std::vector<boost::filesystem::path> getFiles(boost::filesystem::path const& _dirPath, string const& _extentionMask, std::string const& _particularFile)
```

Comment:
Why not just use a `vector` for the extensions?

Reply:
Just remove this param and hardcode expected extensions in getFiles. That would save us some parsing magic.

But set is also good. You can do:
```cpp
#include <set>

auto f(std::set<const char*> s)
{
  return s.size();
}

int main()
{
  return f({".json", ".yml"});
}
```